lib/onc_certification_g10_test_kit/smart_ehr_practitioner_app_group.rb

```diff
@@ -135,14 +136,17 @@ class SmartEHRPractitionerAppGroup < Inferno::TestGroup
           inputs: {
             requested_scopes: { name: :ehr_requested_scopes },
             received_scopes: { name: :ehr_received_scopes }
+          },
```

Comment:
EHR Practitioner App launch shows v1 scopes by default, even when SMARTv2 is selected

This looks like the culprit:
https://github.com/onc-healthit/onc-certification-g10-test-kit/blob/5b4bf5c9505391e9c26c4b1cf4c6fa2b02db34bc/lib/onc_certification_g10_test_kit/smart_ehr_practitioner_app_group.rb#L118-L127

![Screen Shot 2022-08-19 at 4 14 00 PM](https://user-images.githubusercontent.com/41651655/185700013-0ea26fbd-584b-4ebd-b717-0efee1e65fff.png)

Reply:
I updated this PR to make them consistent across all of the launches.